editor/i18n/en/localization.js

```diff
@@ -260,7 +260,7 @@ module.exports = {
             "premultipliedAlpha": "Indicates whether to enable premultiplied alpha.",
             "use_tint": "Indicates whether to use tint effect.",
             "enabled_batch": "Indicates whether enabled batch model",
-            "render_mode": "The render mode of current armature. \n'REALTIME' means REALTIME calculate animation data, support animation blend but low performance. \n'SHARED_CACHE' means precomputation animation data and share data with same armature, high performance and less memery. \n'PRIVATE_CACHE' means precomputation animation data but not share data with other same armature, high performance and more memery.\nAbove two cache mode, not support nest armature, not support 'fade' related interface.",
+            "animation_cache_mode": "REALTIME model, realtime calculate, support all functions of Spine.\nSHARED_CACHE mode, caching and sharing dragonbones animation data, the equivalent of pre baked skeletal animation, have high performance, does not support the action blend and superposition, only supports the start and end events, as for memory, when creating some same bones and the same action of animation, can present advantages of memory, the greater the amount of skeleton, the more obvious advantages, in conclusion SHARED_CACHE mode is suitable for the scene animation, special effects, monster, NPC and so on, can greatly increase the frame rate and reduce memory.\nPRIVATE_CACHE mode, similar to SHARED_CACHE, but not share animation and texture data, so there is no advantage in memory, there is only a performance advantage, when trying to take advantage of caching pattern of high performance, but there is a change of texture, so you can't share the map data, then PRIVATE_CACHE is suitable for you.",
```

Comment:
REALTIME model, the animation data is calculated while running, all features supported.
SHARED_CACHE mode, the animation data is pre-calculated and cached globally, the data is shared between skeleton animation instances. Shared cache mode has great performance advantage, but it has some limitation. It doesn't support action blending and addition and it only supports the start and end events. In the memory usage aspect, as same skeleton animation instances are sharing the same animation data, this mode can save a lot of memory. More instances are sharing, more memory saved. In conclusion SHARED_CACHE mode is suitable for the skeletons which could be replicated, such as effects animation, monster, NPC and so on.\nPRIVATE_CACHE mode, similar to SHARED_CACHE, but the animation data and texture data are not shared between skeleton instances. So there is no advantage in memory compare to REALTIME mode. The PRIVATE_CACHE is suitable when there is no animation blending or addition needed, but the skeleton could change textures of sub bone (e.g. weapon).